packages/geocoder/src/geocoders/pelias.ts

```diff
@@ -11,7 +14,7 @@ export default class PeliasGeocoder extends Geocoder {
    * Generate an autocomplete query specifically for the Pelias API. The
    * `sources` parameter is a Pelias-specific option.
    */
-  getAutocompleteQuery(query) {
+  getAutocompleteQuery(query: AutocompleteQuery): AutocompleteQuery {
```

Comment:
It feels a bit strange to have the input and output here be the same type. I would maybe again add a comment as to what's going on, even if the types are correct.

Reply:
Commented in a3cc58b